src/data/camouflages/skeletal.js

```diff
@@ -20,6 +21,7 @@ const zombies = [
   'Thanatos',
   'Vehement Rage',
   'Harbinger'
```

Comment:
Missing a comma here, probably why the build fails on Vercel 😄 
```suggestion
  'Harbinger',
```